.gitignore

```diff
@@ -1,2 +1,5 @@
 target/
-.idea
\ No newline at end of file
+.idea
```

Comment:
nit: It's often a good idea to leave a blank line at the end of each file so changes like this don't get caught in diffs.